core/src/main/java/com/group5/core/levels/OnlyFloorLevel.java

```diff
@@ -42,6 +43,8 @@ public OnlyFloorLevel(final GameSlice before, final World world) {
     private void createElems(final World world) {
         getElems().add(new FloorTile(world, new Vector2(this.getStartPoint().x / 50, 0)));
         float range = (this.getEndPoint().x - this.getStartPoint().x) / 50;
-        getElems().add(new Obstacle(world, new Vector2((float) (this.getStartPoint().x / 50 + range * Math.random()), 1.2f)));
+        float xPosObstacle = (float) (this.getStartPoint().x / 50 + range * Math.random());
+        getElems().add(new ShootingObstacle(world, new Vector2(xPosObstacle, 1.2f)));
+        getElems().add(new Bullet(world, new Vector2(xPosObstacle, 2.f)));
```

Comment:
So the shootingobstacles should not be put in the GameSlice container but in a container for things that aren't attached to gameslices. When this container is implemented the robot shoot spawn the bullets in there)